### PR TITLE
test(venue):TEMS-61 E2E tests for venue CRUD

### PR DESCRIPTION
**Pull request by junix033101**
## Description

_Provide a description on what was changed/added/fixed in this PR_

## Checklist

_First, create a draft pull request. Then mark the PR as ready for review when all necessary checkbox items has been completed_

- [x] Reviewed own code
- [x] Commented on code that is hard to understand
- [x] Implemented tests for the feature/bugfix
- [x] All **_Required_** GitHub status checks pass
- [ ] The frontend and backend PR previews have been deployed

**Backend only**

- [ ] Added test data for new entity
- [ ] Generated the client api if there are new or deleted endpoints and/or DTOs

### CODE REVIEW

**Review by noice-noise — APPROVED** [review]
💖

**Review by princhcanal — APPROVED** [review]
noice and clean 🧽

File: frontend/src/modules/venues/components/VenuesTable.tsx [hunk 1]
```diff
@@ -186,16 +186,12 @@ export const VenuesTable = ({ refresh }: VenuesTableProps) => {
                   onClick={() => handleView(vn)}
                 >
                   <Td maxW={32}>
-                    <EllipsisText
-                      as='span'
-                      fontWeight='medium'
-                      data-cy='venue-name'
-                    >
+                    <EllipsisText fontWeight='medium' data-cy='venue-name'>
```
Comment (princhcanal) [inline frontend/src/modules/venues/components/VenuesTable.tsx:189]: @noice-noise is it fine that `as='span'` gets removed?

Comment (noice-noise) [reply frontend/src/modules/venues/components/VenuesTable.tsx:189]: Yes, removing the span solves the problem, it's the solution I've provided when jaq raised the issue.

**Pull request merged**